components/common/IdentityForm.vue

```diff
@@ -31,6 +31,21 @@
         <hr class="my-7" />
       </div>
 
+      <NeoField :label="$t('Select Blockchain')">
+        <div class="w-100">
```

Comment:
this `<div>` is redundant

Reply:
yeah I was also thinking about removing it but unfortunatelyi skipped it thanks for the correction